regexes/device/televisions.yml

```diff
@@ -38,12 +38,20 @@ Airties:
     - regex: 'Altech UEC; ?([^);/]+)'
       model: '$1'
 
-# Altus
+# Altus (https://www.altus.com.tr/)
 Altus:
   regex: 'Altus[;,]'
   device: 'tv'
   model: ''
 
+# Amazon
+Amazon:
+  regex: 'Amazon.+AMZ'
```

Comment:
try to avoid greedy regex

'smarttv_\w{3,20}-AMZ-([\w_ -]+)_Build_'
or 
`Amazon;\w{3,20}-AMZ-([\w_ -]+);`

https://regex101.com/r/TffThg/1